<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exportPDF">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>اختر ملف  للتحويل</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrintFile">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>طباعة</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cansel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>الغاء</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseFile">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>اختر ملف</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>تحويل الى pdf</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
